includes/widgets/class-urlslab-media-offloader-widget.php

```diff
@@ -564,15 +597,16 @@ private function schedule_missing_images( array $urls ) {
 
 		foreach ( $urls as $fileid => $url ) {
 			if ( ( urlslab_is_same_domain_url( $url ) && $save_internal ) || $save_external ) {
-				$placeholders[] = '(%s, %s, %s, %s)';
-				array_push( $values, $fileid, $url, Urlslab_Driver::STATUS_NEW, $default_driver );
+				$placeholders[] = '(%s, %s, %s, %s, %s)';
+				array_push( $values, $fileid, $url, isset( $this->parent_urls[ $fileid ] ) ? $this->parent_urls[ $fileid ] : '', Urlslab_Driver::STATUS_NEW, $default_driver );
```

Comment:
`isset( $this->parent_urls[ $fileid ] ) ? $this->parent_urls[ $fileid ] : ''`
can be replaced with:
`$this->parent_urls[ $fileid ]  ?? ''`